flink-runtime-web/web-dashboard/src/app/pages/job/checkpoints/detail/job-checkpoints-detail.component.html

```diff
@@ -26,6 +25,9 @@
     <nz-divider nzType="vertical"></nz-divider>
     <strong>Checkpoint Type:</strong>
     {{ checkPointType }}
+    <nz-divider nzType="vertical"></nz-divider>
```

Comment:
I am curious what you think of the idea that we amend the checkPointType to append (full) or (incremental) for checkpoints so we would see:
`aligned checkpoint (full)`
`aligned checkpoint (incremental)`
`unaligned checkpoint (full)`
`unaligned checkpoint (incremental)`

This would take up less vertical screen real estate.

Reply:
I'd be good with this 👍 
I agreed that horizontal screen space is limited in that tab

I think that the API should have a boolean or Enum.
What do you think between an Enum or boolean flag?

Reply:
![Screenshot 2025-01-07 at 5 27 07 PM](https://github.com/user-attachments/assets/5618469a-84bc-4817-a912-542abde683ad)

Updated to the recommended syntax